src/RA_Dlg_Login.cpp

```diff
@@ -46,9 +51,32 @@ INT_PTR CALLBACK RA_Dlg_Login::RA_Dlg_LoginProc(HWND hDlg, UINT uMsg, WPARAM wPa
                         return TRUE;
                     }
 
+
+                    //URL-escape password so it gets sent to the server properly
+                    //https://stackoverflow.com/questions/154536/encode-decode-urls-in-c
+                    std::ostringstream escaped;
+                    escaped.fill('0');
+                    escaped << std::hex;
+
+                    const std::string &value = ra::Narrow(sPassEntry);
```

Comment:
If you did this because you were afraid `sPassEntry `would be emptied don't be, it'll return a new string. I suggest getting rid `& `for speed `(O(1))`. Shallow copies are usually slow. Plus it gets hard to tell at a glance whether a reference has been invalidated with a static code analysis tool.

Reply:
`ra::Narrow` returns a `std::string` object created inside the method. Depending on how the compiler optimizes that, the variable may go out of scope before you're done with it. Recommend removing the reference: 
```
const std::string sValue = ra::Narrow(sPassEntry);
```
Also note the Hungarian prefix. Please follow the coding styles outlined in the [CONTRIBUTING.md](https://github.com/RetroAchievements/RAIntegration/blob/master/CONTRIBUTING.md) file